libtsuba/src/RDGManifest.cpp

```diff
@@ -263,15 +217,16 @@ RDGManifest::FileNames() {
   for (auto i = 0U; i < num_hosts(); ++i) {
     // All other file names are directory-local, so we pass an empty
     // directory instead of handle.impl_->rdg_manifest.path for the partition files
-    fnames.emplace(PartitionFileName(view_type_, i, version()));
+    fnames.emplace(PartitionFileName(view_specifier(), i, version()));
 
-    auto header_res =
-        RDGPartHeader::Make(PartitionFileName(dir(), i, version()));
+    auto header_uri = KATANA_CHECKED(katana::Uri::Make(fmt::format(
+        "{}/{}", dir(), PartitionFileName(view_specifier(), i, version()))));
+    auto header_res = RDGPartHeader::Make(header_uri);
 
     if (!header_res) {
       KATANA_LOG_DEBUG(
```

Comment:
If this is an error, should we be returning `KATANA_ERROR` instead of a doing a debug log?

Reply:
I think either approach is fine. I would prefer to simply log the error as the user will presumably get an error if he later attempts a RDGPartHeader::Make.